pkg/v1/tkg/test/tkgctl/tkgs_cc/tkgs_cc_workload_cluster_test.go

```diff
@@ -100,12 +101,13 @@ var _ = Describe("TKGS ClusterClass based workload cluster tests", func() {
 		})
 
 		It("should successfully create a cluster and verify successful addons reconciliation", func() {
-			shared.CheckTKGSAddons(ctx, tkgctlClient, svClusterName, clusterName, namespace, e2eConfig.TKGSKubeconfigPath, constants.InfrastructureProviderTkgs, false)
 			Expect(err).ToNot(HaveOccurred())
+			shared.CheckTKGSAddons(ctx, tkgctlClient, svClusterName, clusterName, namespace, e2eConfig.TKGSKubeconfigPath, constants.InfrastructureProviderTkgs, false)
```

Comment:
we can probably skip checking for addons if there was an error in cluster creation, so lets move the line further down.